uaclient/messages/__init__.py

```diff
@@ -1426,6 +1426,8 @@ class TxtColor:
 REALTIME_NVIDIA_DESCRIPTION = t.gettext(
     "RT kernel optimized for NVIDIA Tegra platform"
 )
+REALTIME_RASPI_TITLE = t.gettext("Real-time Raspberry Pi Kernel")
+REALTIME_RASPI_DESCRIPTION = t.gettext("RT kernel optimized for Raspberry Pi")
```

Comment:
@EdoardoBarbieriCanonical Are these a reasonable title and description of the raspberry pi variant? Should we mention a specific version or versions of the rpi?

Also, I have named the variant "raspi", so the command would be `pro enable realtime-kernel --variant=raspi`.

We need to decide all of these things now if we want this to even be a possibility in April.

Reply:
Thanks for the PR. Can we go with?

- Title: "Raspberry Pi Real-time for Pi5/Pi4"
- Description: "24.04 Real-time kernel optimised for Raspberry Pi"
- Command: `pro enable realtime-kernel --variant=rpi`

Reply:
Added. Thanks @EdoardoBarbieriCanonical!